fluent-react/package.json

```diff
@@ -72,9 +73,8 @@
     "@fluent/bundle": "file:../fluent-bundle",
     "@types/react": "^16.9.22",
     "@types/react-dom": "^16.9.5",
-    "babel-jest": "^24.8.0",
-    "jest": "^24.8.0",
-    "prettier": "^1.18.2",
```

Comment:
Question: Why is this unneeded? Is prettier configured elsewhere?

Reply:
It's no longer required by Jest, which was its only user: facebook/jest#11400

It wasn't actually used to format code in `fluent-react/`.